app/models/manageiq/providers/hawkular/inventory/parser/availability_updates.rb

```diff
@@ -0,0 +1,22 @@
+module ManageIQ::Providers
+  class Hawkular::Inventory::Parser::AvailabilityUpdates < ManagerRefresh::Inventory::Parser
+    def parse
+      fetch_server_availabilities
+      fetch_deployment_availabilities
+    end
+
+    def fetch_server_availabilities
```

Comment:
it looks like there's an opportunity here for code reuse.
because this method and the one below look exactly the same